fasort.sh

```diff
@@ -2,21 +2,21 @@
 # Author: Noxy (a dhole who likes paws)
 # This script will find and move any FA files in the working directory and move them to artist folders under $SORTDIR.
 
+set -euo pipefail
```

Comment:
This is always useful, from the bash docs:

https://www.gnu.org/software/bash/manual/html_node/The-Set-Builtin.html

> -e
>   Exit immediately if a pipeline (see Pipelines), which may consist of a single simple command (see Simple 
>   Commands), a list (see Lists), or a compound command (see Compound Commands) returns a non-zero status.

>-u
Treat unset variables and parameters other than the special parameters ‘@’ or ‘*’ as an error when performing parameter expansion. An error message will be written to the standard error, and a non-interactive shell will exit.

>-o pipefail
If set, the return value of a pipeline is the value of the last (rightmost) command to exit with a non-zero status, or zero if all commands in the pipeline exit successfully. This option is disabled by default.

Basically, if you set this, all exit codes for commands must be 0 or you have to do error handling otherwise the script will exit, same for unset variables.